src/bp-core/bp-core-template.php

```diff
@@ -4590,6 +4590,101 @@ function bb_user_has_access_upload_video( $group_id = 0, $user_id = 0, $forum_id
 	}
 
 	return false;
+}
+
+/**
+ * Get user followers count.
+ *
+ * @param int|null $user_id user id to get followers count. If user id is null then get current logged-in user id.
+ *                 Default false.
+ *
+ * @since BuddyBoss [BBVERSION]
+ *
+ * @return string
+ */
+function bb_get_followers_count( $user_id = false ) {
+
+	if ( ! function_exists( 'bp_is_active' ) && ! function_exists( 'bp_is_activity_follow_active' ) ) {
+		return;
+	}
+
+	$is_follow_active = bp_is_active( 'activity' ) && function_exists( 'bp_is_activity_follow_active' ) && bp_is_activity_follow_active();
 
+	if ( false === $user_id ) {
+		$user_id = bp_displayed_user_id();
+	}
+
+	if ( $is_follow_active && is_user_logged_in() ) {
+		$total_followers = 0;
+		$follower_ids    = bp_get_follower_ids( array( 'user_id' => $user_id ) );
+
+		if ( ! empty( $follower_ids ) ) {
+			$total_followers = count( explode( ',', $follower_ids ) );
+		}
+
+		if ( 0 === $total_followers ) {
+			$followers = __( '<b>0</b> followers', 'buddyboss' );
+		} elseif ( 1 === $total_followers ) {
+			$followers = __( '<b>1</b> follower', 'buddyboss' );
+		} else {
+			$followers = sprintf(
+				/* translators: Total followers count. */
+				__( '<b>%s</b> followers', 'buddyboss' ),
+				$total_followers
+			);
+		}
+		?>
+
+		<div class="followers-wrap"><?php echo wp_kses_post( $followers ); ?></div>
+		<?php
+	}
+}
```

Comment:
@milanhirapra Move this function to their respective functions file instead of core.

Reply:
done